controls/eventhub.sp

```diff
@@ -9,8 +9,10 @@ benchmark "eventhub" {
   description = "This benchmark provides a set of controls that detect Terraform Azure Event Hub resources deviating from security best practices."
 
   children = [
+    control.event_hub_namespace_zone_redundant,
```

Comment:
Following the other control namings it should be `eventhub_namespace_zone_redundant`